README.md

```diff
@@ -61,7 +60,7 @@ sudo docker run --gpus all -it --shm-size=8g --name functionary -v ${PWD}/functi
 ```
 
 ### Call Real Python Function
-To call the real python function, get the result and extract the result to respond, you can use [chatlab](https://github.com/rgbkrk/chatlab).
```

Comment:
I'd love to figure out how I can help this run with `chatlab==1.0.0`. It also seems like we probably should have both a vanilla Python version using the `chatlab.FunctionRegistry` as well as the notebook variant that can call `await chat("Some user request")`